config/routes.rb

```diff
@@ -14,6 +14,7 @@
   get '/dashboards' => 'dashboards#index'
   post '/dashboards/build' => 'dashboards#build', as: :dashboard_build
   get '/dashboards/export/:csv_type' => 'dashboards#export', as: :dashboard_export, defaults: { format: 'csv' }
+  get '/dashboards/export/version/:number' => 'dashboards#export_version', as: :dashboard_export_version, defaults: { format: 'csv' }
```

Comment:
What do you think of changing this to `get '/dashboards/export/institutions/:number'`?
Keeps it in line with the other `export` URL.

Reply:
Sounds good